automation/alpha-archived-metadata-file-upload-check.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VAULT_TOKEN=$(cat /etc/vault-token-dsde)
+
+# Get a workflow which completed at least 40 minutes ago and whose metadata is archived
+# (archive-delay in alpha is set to 30 minutes. Hence adding 10 more minutes should give Cromwell enough time to
+# archive such a workflow)
+ARCHIVED_WORKFLOW_RESULT=$(
+  docker run --rm \
+     -e VAULT_TOKEN=$VAULT_TOKEN \
+     broadinstitute/dsde-toolbox mysql-connect.sh -p firecloud -a cromwell1 -e alpha \
+     "SELECT WORKFLOW_EXECUTION_UUID, ROOT_WORKFLOW_EXECUTION_UUID FROM cromwell.WORKFLOW_METADATA_SUMMARY_ENTRY WHERE METADATA_ARCHIVE_STATUS = 'Archived' AND END_TIMESTAMP <= (NOW() - INTERVAL 40 minute) ORDER BY END_TIMESTAMP DESC LIMIT 1;"\
```

Comment:
Once the archiver catches up, it might be a good idea to check the time stamp is actually **recent** enough (eg within the last day) rather than **old** enough. Eg if the archiver stalls, this test will keep passing because it will just keep finding the most recent successfully archived entry

Reply:
That makes sense. I can create a ticket to change this query logic to search for workflows whose `END_TIMESTAMP <= (NOW() - INTERVAL 40 minute) AND END_TIMESTAMP > (NOW() - INTERVAL 1 DAY)` once the archiver has caught up and after we change the `archive-delay` to 30 minutes. How does that sound?

Reply:
Sounds good to me

Reply:
Created https://broadworkbench.atlassian.net/browse/BW-657